docs/how-to-guides/how-to-config-layers.md

```diff
@@ -0,0 +1,76 @@
+# How to Config Layers
```

Comment:
Let's use the full word, "configure". Also, it looks like our docs generally don't use title case, see for example https://juju.is/docs/sdk/how-to and https://documentation.ubuntu.com/lxd/en/latest/howto/, so I suggest we try to be consistent with that.

```suggestion
# How to configure layers
```

Reply:
Also, can we shorten the URL, Currently it's:

```
/en/419/how-to-guides/how-to-config-layers/
```

Ideally we could do either of the following, to avoid the duplication:

```
/en/419/howto/configure-layers/
/en/419/how-to-configure-layers/
/en/419/configure-layers/  # looks like Juju mostly does it this way
```

Reply:
I have done some research on this:

1. Juju docs isn't using the starter pack, but from discourse, that's why the behaviour is a bit different, doesn't have the folder name as part of the URL.
2. It seems other Canonical documentation that uses the starter pack, for example, LXD (using it as an example because LXD is also using markdown rather than rst like us), does have the folder name as part of the URL, but it's shorter.
3. In rare cases, a page can be put into a folder with no folder name, but that's only because the file actually isn't in the folder, but put in by some other directives. For example, [this page from lxd](https://documentation.ubuntu.com/lxd/en/latest/getting_started/) shows up in the folder with no folder name in the URL (this is an exception), only because the file is directly in the root of ["docs"](https://github.com/canonical/lxd/tree/main/doc). I intentionally made the second link to the folder rather than the file to show that this might be a bad practice, and hard to manage since we might end up with many files at the root level not knowing which belongs to which.

So, my conclusion is like the first example you've given `/en/419/howto/configure-layers/`, that is, with a short folder name, and a shorter article name. Maybe we should try to put as less articles as possible at the root level of docs to make it manageable (the current LXD repo is hard to maintain in my opinion.)